Lib/test/test_deque.py

```diff
@@ -517,7 +517,7 @@ def test_remove(self):
         for match in (True, False):
             d = deque(['ab'])
             d.extend([MutateCmp(d, match), 'c'])
-            self.assertRaises(IndexError, d.remove, 'c')
+            self.assertRaises((IndexError, RuntimeError), d.remove, 'c')
```

Comment:
Wouldn't this now always raise `RuntimeError`? If so, remove the `IndexError`.

Reply:
Corrected in 6317c78